Cargo.toml

```diff
@@ -22,6 +22,5 @@ compiler_builtins = { version = "0.1", optional = true }
 
 [features]
 default = []
-no_std = []
```

Comment:
This is fast! If you want to remove this feature, maybe you want to remove some corresponding comments in lib.rs mod comments too?